sessions/testing.qmd

```diff
@@ -151,7 +152,7 @@ task, and pushing to GitHub.
 TODO: Reading text
 :::
 
-## Exercise: TODO: Add title
+## Exercise: TODO: Create unit tests for the remaining functions in your package
```

Comment:
This is a bit too generic. It ideally would be a clear, explicit aim. If the first exercise is a code reading exercise, this one could be to create another unit test to check for some condition. Or even include a first task to ask them to select which condition to test for and then test it.